src/components/Desk.tsx

```diff
@@ -0,0 +1,50 @@
+import { useYDoc } from "zustand-yjs";
+import { YJS_ROOM } from "../constants";
+import { LetterSharedData, LetterInterface } from "../types";
+import { Letters } from "../data/letters";
+import { useEffect, useState } from "react";
+import { LetterFormButton } from "./LetterForm";
+import { Letter } from "./Letter";
+import { ShuffleIcon, ResetIcon, ViewGridIcon } from "@radix-ui/react-icons";
+import { connectDoc } from "../utils/yjs";
+import { SubmitLetterMetadata } from "../App";
+
+export function Desk() {
+  const yDoc = useYDoc(YJS_ROOM, connectDoc);
+  const sharedMap = yDoc.getMap<LetterSharedData>("shared");
+  const [letters, setLetters] = useState<LetterInterface[] | undefined>();
+
+  useEffect(() => {
+    setLetters([
+      ...Letters,
+      {
+        ...SubmitLetterMetadata,
+        ctaContent: <LetterFormButton letter={Letters[0]}></LetterFormButton>,
+      },
+    ]);
+  }, []);
```

Comment:
Why don't we do this statically in the Letters const? I'm pretty sure you can include JSX in regular variables

Reply:
oh yeah let me do that -- i did this orignally because i wanted to be able to pass in the letter, so you could open a modal for any of the letters to expand, read in-depth etc. but ic ouldnt get that to work with this dialog module 💀 might swap out in future if we want that for other letters

Reply:
oh actually i remember i needed sometihng w/o the cta content for rendering the editable view